src/components/SelectionList/BaseSelectionList.js

```diff
@@ -166,7 +165,20 @@ function BaseSelectionList({
         }
 
         listRef.current.scrollToLocation({sectionIndex: adjustedSectionIndex, itemIndex, animated, viewOffset: variables.contentHeaderHeight});
-    };
+
+        // If this function changes, it causes `useArrowKeyFocusManager` to fire `onFocusedIndexChange`,
+        // making the list scroll back to the focused index when the keyboard disappears. If we don't disable
+        // dependencies here, we would need to make sure that the `sections` is stable in every usage of this component.
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({
+        maxIndex: flattenedSections.allOptions.length - 1,
+        onFocusedIndexChange: scrollToIndex,
+        initialFocusedIndex: _.findIndex(flattenedSections.allOptions, (option) => option.keyForList === initiallyFocusedOptionKey),
+        disabledIndexes: flattenedSections.disabledOptionsIndexes,
+        isActive: !disableKeyboardShortcuts,
+    });
```

Comment:
Switched from `<ArrowKeyFocusManager />` to `useArrowKeyFocusManager`, no change of behavior